src/server/config.js

```diff
@@ -5,23 +5,36 @@ import cjson from 'cjson';
 // avoid ESLint errors
 const logger = console;
 
+// Tries to load a .babelrc and returns the parsed object if successful
+function loadBabelConfig(babelConfigPath) {
+  let config;
+  if (fs.existsSync(babelConfigPath)) {
+    const content = fs.readFileSync(babelConfigPath, 'utf-8');
+    try {
+      config = cjson.parse(content);
+      config.babelrc = false;
```

Comment:
What's the meaning of this?
Can't we use this for everything since we load and pass the babelrc manually anyway.